vizro-core/src/vizro/models/_components/button.py

```diff
@@ -29,12 +36,27 @@ class Button(VizroBaseModel):
         PlainSerializer(lambda x: x[0].actions),
         Field(default=[]),
     ]
+    extra: SkipJsonSchema[
+        Annotated[
+            dict[str, Any],
+            Field(
+                default={},
+                description="""Extra keyword arguments that are passed to `dbc.Button` and overwrite any
+            defaults chosen by the Vizro team. This may have unexpected behavior if the defaults change.
+            Visit the [dbc documentation](https://dash-bootstrap-components.opensource.faculty.ai/docs/components/button/)
+            to see all available arguments. [Not part of the official Vizro schema](../explanation/schema.md) and may
+            not be supported in the future. Defaults to `{}`.""",
+            ),
+        ]
+    ]
 
     @_log_call
     def build(self):
-        return dbc.Button(
-            id=self.id,
-            children=self.text,
-            href=get_relative_path(self.href) if self.href.startswith("/") else self.href,
-            target="_top",
-        )
+        defaults = {
+            "id": self.id,
+            "children": self.text,
+            "href": get_relative_path(self.href) if self.href.startswith("/") else self.href,
+            "target": "_top",
+        }
```

Comment:
This is just an alternative and I have no strong preferences. Ignore the comment if you prefer `{}` over `dict()`. 😄 
```suggestion
        defaults = dict(
            id=self.id,
            children=self.text,
            href=get_relative_path(self.href) if self.href.startswith("/") else self.href,
            target="_top",
        )
```

Reply:
Indeed, so will leave it at the old thing :)

Reply:
Not auto revert, but complain I htink